### ISSUE TITLE
Your conference is currently being created...

### ISSUE THREAD

**Issue by PocoCraft** [jitsi/jitsi-meet#300]
Hello. I just installed Jitsi-Meet onto one of my VPS' and when I create a room and attempt to share the invite link I get "Your conference is currently being created..."
Is there something that I am missing?
I've tried reinstalling using both 'talk', and 'talk.mydomainhere.mytldhere' as the hostname..
I cannot find anything on any of the Google results that has resolved my problem.

**Comment by Nodeswitch** [comment]
I'm also experiencing the same. I installed using the quick-install method.

**Comment by paweldomas** [comment]
What do you see in the logs located at /var/log/jitsi ?

**Comment by Nodeswitch** [comment]
Here's <a href="http://pastebin.com/pQ2fgh92">jicofo.log</a> and <a href="http://pastebin.com/h7Y2eRnc">jvb.log</a>.

I rebooted the server several times, so there will be repeat entries. Early on in jicofo.log is lists network connectivity issues, but this doesn't occur again.

**Comment by PocoCraft** [comment]
I see jifco.log and jvb.log

**Comment by PocoCraft** [comment]
Inside both I see Java looking errors complaining about "Connection refused"

**Comment by PocoCraft** [comment]
Tell me if I need to upload my logs as well.

**Comment by paweldomas** [comment]
>Here's jicofo.log and jvb.log.
>
>I rebooted the server several times, so there will be repeat entries. Early on in jicofo.log is lists >network connectivity issues, but this doesn't occur again.
>
>In case it was a communication error, I opened up all ports to the VM temporarily to test and that >didn't change anything.

The logs look ok after restart. Can you also copy JavaScript console logs from Chrome after trying to invite others ? ctrl+shift+j

**Comment by PocoCraft** [comment]
http://gyazo.com/7f73de3ea7ede040b455338a8f734bdf

I took out my domain and TLD only.

**Comment by Nodeswitch** [comment]
<a href="http://pastebin.com/HWsTt6fT">Here goes</a>. Differs from the above.

**Comment by paweldomas** [comment]
>http://gyazo.com/7f73de3ea7ede040b455338a8f734bdf
>
>I took out my domain and TLD only.

Prosody bosh runs on different domain than the website ?

**Comment by paweldomas** [comment]
>Here goes. Differs from the above.
>
>On browsing around "Failed to load resource: net::ERR_CONNECTION_REFUSED" seems to be >linked to ad blocking plugins in some cases, but I've got a fresh install of Chrome going here and >have tested from another machine as well where I've encountered the same issue.

Prosody is misconfigured ? Can you show also /etc/prosody/prosody.cfg.lua and /etc/prosody/conf.avail.hangout.cfg.lua ?

**Comment by Nodeswitch** [comment]
<a href="http://pastebin.com/rpe74Ufa">hangout.cfg.lua</a> and <a href="http://pastebin.com/qXm689y4">prosody.cfg.lua</a>.

Thanks for helping with this!

**Comment by PocoCraft** [comment]
Both of those files I have not edited myself. I checked the ones that Nodeswitch posted, and they match mine, except the obviously different parts, such as the domain, TLD, secret key, etc. Again, if you need me to upload mine as well ask me.

**Comment by PocoCraft** [comment]
I really appareciate the timely responses and the help you are providing!

**Comment by paweldomas** [comment]
Both look ok. I need also /etc/nginx/sites-available/hangouts

Btw. what URL do you exactly enter in the browser ?

What happens if you open https://hangout/http-bind ?

**Comment by PocoCraft** [comment]
http://pastebin.com/U4fZ23sQ

**Comment by PocoCraft** [comment]
I enter talk.domain.tld:8080
It loads exactly the same way it did when I was running on the default ports, so I changed it to a different port without SSL since I run an Apache webserver on ports 80 and 443 as well, which cause conflicts when both are trying to run.

**Comment by Nodeswitch** [comment]
<a href="http://pastebin.com/LhbNtYCV">Here's</a> the Nginx conf file. 

I get "Unable to connect" when I try https://hangout/http-bind. I checked this out last night before you replied as I saw the BOSH entry in the config, but I can't locate an actual file or folder called http-bind on the system. Should this be something that I can locate?

**Comment by PocoCraft** [comment]
Does anyone know what our problem might be with the information we have provided?

**Comment by jaysig112** [comment]
Has anyone found a solution for this problem I tried a clean install on unbuntu 14 and I am getting the Conference is being created error. Also I am getting the connection refused in the logs.

**Comment by damencho** [comment]
Default prosody packages, under 14.04 and installing jitsi-meet the quick
way by just doing # apt-get install jitsi-meet and no custom modifications
should work. Definitely you need to fix the connection refused problems you
face. Do you have any custom settings or packages for prosody or nginx not
coming from ubuntu repo?

**Comment by akuckartz** [comment]
I am seeing this issue ("Die Konferenz wird erstellt...") on https://beta.meet.jit.si

Another issue is that I am unable to give myself an (account) name.

**Comment by damencho** [comment]
If you see it with beta, then it can be something with your browser. Can
you check your browser console, are there any errors?

On Fri, Oct 30, 2015 at 12:12 PM, Andreas Kuckartz <notifications@github.com

**Comment by jaysig112** [comment]
I started a brand new unbuntu vm from scratch and all I did was set a hostname on the machine. Once that is done I followed the quick install guide and I got both of those problems. I tried it on the unbuntu server and desktop.

**Comment by damencho** [comment]
Can you upload your jicofo and jvb logs somewhere so we can take a look?
What are the connection refused logs? Seems your components jicofo or jvb
cannot connect to prosody on port 5347 on localhost, it can be some
firewall or misconfiguration.

**Comment by jaysig112** [comment]
here is most of the jvb logs http://pastebin.com/R6nBUTTD
here is the jifcofo logs http://pastebin.com/iRWk0UQr

**Comment by jaysig112** [comment]
After taking a look at the logs did you happen to find anything?

**Comment by damencho** [comment]
What I see in jicofo logs is that jicofo cannot connect to localhost port
5347 port. And this is the same error and for jvb, at least the logs start
with that.

**Comment by jaysig112** [comment]
I used ufw allow 5347 to make sure those were open I don't actually know why it wont allow it to connect to that port.

**Comment by jaysig112** [comment]
Would this be a misconfiguration because of the Ubuntu iso I have or the quick-install because I didn't touch anything on the system after doing quick-install.

**Comment by damencho** [comment]
By default ubuntu-server comes with no firewall configured. And there is
nothing in the configuring of our debian packages that care for firewall,
its user issue to work on that.

**Comment by rabiajavaid** [comment]
Same issue? how to resolve this. i cant telnet port 5347

**Comment by jaysig112** [comment]
My issue was the hostname I was trying to use make sure you have it all configured correctly if you have a hostname you are trying to use.

**Comment by rabiajavaid** [comment]
Issue resolved. There were jre permission issues, ip tables issues, hostname and nginx file configuration issues.
